roamance/src/components/social/comment/comment-form.tsx

```
@@ -0,0 +1,320 @@
+          </motion.div>
+
+          <div className="flex-1">
+            <textarea
+              value={text}
+              onChange={handleTextChange}
+              onClick={() => setIsExpanded(true)}
+              placeholder={placeholder}
+              rows={isExpanded ? 2 : 1}
+              autoFocus={autoFocus}
+              className="w-full p-2.5 rounded-xl bg-white/80 dark:bg-gray-800/40 backdrop-blur-sm border border-gray-100/80 dark:border-gray-700/50 resize-none focus:outline-none focus:ring-2 focus:ring-ocean/40 text-gray-800 dark:text-gray-200 placeholder-gray-400 dark:placeholder-gray-500 transition-all duration-200 text-sm shadow-inner-soft"
+            />
```

Biome (1.9.4)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)